src/api/appUsers.js

```diff
@@ -1,23 +1,69 @@
-import BaseAPI from './base';
-import http from '../utils/http';
+import { BaseAPI } from './base';
+import { http } from '../utils/http';
 
-export default class AppUsersAPI extends BaseAPI {
-  retrieve(userId, auth) {
-    const url = this.getFullURL('appUsers', userId);
+
+/**
+ * Init API properties
+ * @typedef InitProps
+ */
+
+/**
+ * @class AppUsersAPI
+ * @extends BaseAPI
+ */
+export class AppUsersAPI extends BaseAPI {
+
+  /**
+   * Initializes the conversation for a user
+   * @param  {InitProps} props
+   * @param  {AuthCredentials} auth
+   * @return {APIResponse}
+   */
+  init(props, auth) {
```

Comment:
@alavers  Yeah I was on the fence on that... I wasn't sure of how it would/should be used.. on the browser side I'd expect the auth props to always be the same since you act on a single user (but then logging in and out means reinstanciate the core instead of keeping one reference around) and on node I thought it would me be used as a single instance for everything.